dex/src/test/scala/com/wavesplatform/matcher/market/OrderBookActorSpecification.scala

```diff
@@ -257,5 +272,64 @@ class OrderBookActorSpecification extends MatcherSpec("OrderBookActor") with NTP
       orderBook ! wrap(2, Canceled(buyOrder.assetPair, buyOrder.id()))
       tp.expectMsgType[OrderCanceled]
     }
+
+    val switchRulesTest = NonEmptyList(
+      MatchingRules.Default,
+      List(
+        MatchingRules(4, OrderBook.TickSize.Enabled(100)),
+        MatchingRules(10, OrderBook.TickSize.Enabled(300))
+      )
+    )
+    "rules are switched" in obcTestWithMatchingRules(switchRulesTest) { (pair, orderBook, tp) =>
+      val buyOrder = buy(pair, 100000000, 0.0000041)
+      (0 to 17).foreach { i =>
+        orderBook ! wrap(i, buyOrder)
+        tp.expectMsgType[OrderAdded]
+      }
+
+      eventually {
+        val bids = obc.get(pair).bids
+        bids.size shouldBe 3
+
+        val l41 = bids.head
```

Comment:
I didn't get the idea of naming
If you mean price level, why not smth like `priceLevel41/priceLevel40/priceLevel30`?

Reply:
Sorry, didn't distinguish lowercase "L" and 1 :(